guides/user/pipeline-expressions/index.md

```diff
@@ -24,33 +24,24 @@ I can now set up my pipeline to take a `command` parameter, when I manually exec
 
 ![](2.png)
 
-When the pipeline runs, it will substitute the value I have entered for `command` with the value `sleep 40000` that I have entered. 
+When the pipeline runs, it resolves the expression to what I’ve entered for `command`.
```

Comment:
Everything I've written has been in 2nd person so far, but I see the rest of this guide is in 1st person. Which one should we pick for the sake of consistency?

Reply:
For *person*, I've left this unchanged for now. Where the original writer used first person I kept it that way in this pass. But you make a good point&mdash;it should be consistent throughout, and I'll make it so. But I'll wait till the next pass on this page to updated it here.